inst/templates/ad_adlb.R

```diff
@@ -128,32 +128,26 @@ adlb <- adlb %>%
     by_vars = vars(STUDYID, USUBJID, !!!adsl_vars, DOMAIN, VISIT, VISITNUM, ADT, ADY),
     set_values_to = vars(
       PARAMCD = "BASO",
-      PARAM = "Basophils Abs (GI/L)",
+      PARAM = "Basophils Abs (10^9/L)",
       PARAMN = 6,
       DTYPE = "CALCULATION",
       PARCAT1 = "HEMATOLOGY"
     ),
     get_unit_expr = extract_unit(PARAM),
-    wbc_code = "WBC",
-    wbc_unit = "GI/L",
-    diff_code = "BASOLE",
-    diff_type = "fraction"
+    diff_code = "BASOLE"
   ) %>%
   # Derive absolute Lymphocytes
   derive_param_wbc_abs(
     by_vars = vars(STUDYID, USUBJID, !!!adsl_vars, DOMAIN, VISIT, VISITNUM, ADT, ADY),
     set_values_to = vars(
       PARAMCD = "LYMPH",
-      PARAM = "Lymphocytes Abs (GI/L)",
+      PARAM = "Lymphocytes Abs (10^9/L)",
       PARAMN = 25,
       DTYPE = "CALCULATION",
       PARCAT1 = "HEMATOLOGY"
     ),
     get_unit_expr = extract_unit(PARAM),
-    wbc_code = "WBC",
-    wbc_unit = "GI/L",
-    diff_code = "LYMPHLE",
-    diff_type = "fraction"
+    diff_code = "LYMPHLE"
```

Comment:
Typically do we specify the `ANRLO` and `ANRHI` for those derived records? Currently, the grades are not calculated for derived records because those variables are not populated.

Reply:
We do specify ranges in Roche, but not sure if all companies would use the same ranges, or even populate them. I thought it was better to leave as them blank.